debian/control

```diff
@@ -103,218 +110,13 @@ Description: Go programming language compiler - metapackage
  This package is a metapackage that, when installed, guarantees
  that (most of) a full Go development environment is installed.
 
-Package: golang-go-linux-amd64
-Architecture: i386 amd64 armel armhf
-Depends: ${shlibs:Depends}, ${misc:Depends}, ${perl:Depends}
-Breaks: golang-go (<< 2:1.1-2)
-Replaces: golang-go (<< 2:1.1-2)
-Description: Go standard library compiled for linux_amd64
- This package contains the Go standard library,
- compiled for linux_amd64.
+Package: golang-pkg
+Architecture: amd64 arm64 armel armhf i386 ppc64 ppc64el
+Depends: ${misc:Depends}, ${perl:Depends}, ${shlibs:Depends}
+Breaks: golang-go (<< 2:1.1-2), golang-go-${go:Hostos}-${go:Hostarch}
+Replaces: golang-go (<< 2:1.1-2), golang-go-${go:Hostos}-${go:Hostarch}
```

Comment:
I'm a little bit scared to say that this `Breaks`/`Replaces` might actually need to list ALL the OS/ARCH combinations we previously supported, but I have no idea.  Technically, the old `golang-go-XXX-YYY` packages should be installable along side this (just not the one for `GOHOSTOS`-`GOHOSTARCH`), but I don't know what kind of problems that'll cause.